src/middlewares.js

```diff
@@ -424,6 +425,54 @@ export function promiseEnforceMasterKeyAccess(request) {
   return Promise.resolve();
 }
 
+export const addRateLimit = (route, id) => {
+  const config = Config.get(id);
+  const express = config.expressApp;
+  addLimitForRoute(express, route);
+};
+
+export const handleRateLimit = (api, rateLimitOptions) => {
+  const options = Array.isArray(rateLimitOptions) ? rateLimitOptions : [rateLimitOptions];
+  for (const route of options) {
+    addLimitForRoute(api, route);
+  }
+};
+
+const addLimitForRoute = (api, route) => {
+  api.use(
+    route.path ?? '*',
+    rateLimit({
+      windowMs: route.windowMs ?? 60000,
```

Comment:
It seems by default, all routes are limited to 3 requests within 1 minute - do these default values make practical sense?

Reply:
I'm note sure, what do you think would would be a practical limit?

Reply:
If we plan to enable rate limiting by default in a future version of Parse Server, these limits should fit a large range of use cases. The limits depend on how tightly resources are proportioned to normal traffic, which can cannot know.

If we choose a random number for all routes combined, it may be:
- too high for hobby projects, because they are dimensioned for lower traffic
- too low for business projects, because their normal traffic is higher

Maybe 1000 rps for all routes combined is a number that makes sense for a broad number of applications, but that's only a guess. Serious DDoS attacks are usually in the million rps, there was a 46M rps attack just this year.

Reply:
1000 requests from a single IP That seems a bit high in my opinion

Reply:
Or we can just remove the default values and require the configuration to explicitly define them all.

Reply:
Yeah that sounds like a good idea

Reply:
Alright let's go that then